### DISCUSSION TITLE
Convention sur la présentation des figures

### DISCUSSION THREAD

**Discussion by aushay** [BIO6032/2022-SEIR-reproduction#8] [General]
Est-ce qu'on présente les figures en noir et blanc/couleur ?

- [ ] Noir et blanc 
- [x] Couleur 

Quel choix fait-on pour l'emplacement des légendes ?

**Comment by Marine1719** [comment]
Je penche aussi pour couleurs, surtout que c'est nécessaire pour les graphiques avec plusieurs R0, sinon impossible de les distinguer. Pour les légendes il faut trouver où se trouvent les recommandations
